docs/user_guide.md

```diff
@@ -248,3 +249,75 @@ If you are using a self-signed cert, make sure to copy the CA cert into ```/etc/
+If error occurred, you can click the **'View Log'** link to open the log viewer to check the related logs.
+![browse project](img/log_viewer.png)
+
+If process is successfully completed, a result bar chart is created. The width of the different colors are used to indicate the percent of features with different severity vulnerabilities found in the overall features. 
+* **Color Red:** High level vulnerabilities
+* **Color Orange:** Medium level vulnerabilities
+* **Color Yellow:** Low level vulnerabilities
+* **Color Grey:** Unknown level vulnerabilities
+* **Color Green:** None vulnerabilities
+![browse project](img/bar_chart.png)
+
+Move cursor over the bar chart, a tooltips with summary report will displayed. Besides telling you the total features with vulnerabilities and the total count of features in the scanned tag, the report also lists the counts of features with different severity vulnerabilities. The completion datetime of the last analysis process is also appended at the footer of the tooltips.
+![browse project](img/summary_tooltip.png)
+
+Click the tag name link, the detail page will be opened. Besides more detailed information about the tag itself, all the vulnerabilities found in last analysis process will be listed here with the related information. You can order or filter the list by columns.
+![browse project](img/tag_detail.png)
+
+**NOTES: You can trigger the vulnerability analysis action for tag at anytime you want as long as the status is not 'Queued' or 'Scanning'** 
+
+**Scan All**
+
+Go to **'Vulnerability'** tab of **'Configuration'** section under **'Administration'**, you'll find the **'SCAN NOW'** button. Click that button will start the analysis process for all the existing images. 
+
+**NOTES: The process above is executed via multiple concurrent asynchronous tasks and then no guarantee to the order of executing or the results being returned.** 
+![browse project](img/scan_all.png)
+
+To avoid frequent triggering this 'heavy' action, the availability of the button will be restricted. It can be only triggered once in a specified duration. The next available time will be displayed as tips aside the button.
+![browse project](img/scan_all2.png)
+
+**Scan via Policy** 
+
+You can set related policies to control the vulnerability analysis process. Currently, two options are provided for you:
+* **None:** No policy is activated
+* **Daily:** Daily policy is activated, that means an analysis job is scheduled to execute at the specified time everyday. The scheduled job will scan all the images in Harbor.
+![browse project](img/scan_policy.png)
+
+**NOTES: Once the scheduled job is executed, the scanning completion datatimes of all images will be updated. Please be aware that the completion datatimes of the images might be different because the execution of analysis for each image may be arranged in different turns as the restriction of job queue.**

```

Comment:
+**NOTES: Once the scheduled job is executed, the scanning completion time of all images will be updated accordingly. Please be aware that the completion time of the images may be different because the execution of analysis for each image may be carried out at different time.**